index.js

```diff
@@ -81,6 +81,7 @@ function initStorage(loadhook) {
         }
       }
     };
+    var tx = db.transaction('history');
```

Comment:
This redefines `tx` which is harmless but not good style. Can you just remove the `var` ?